MapleServer2/Managers/TrophyManager.cs

```diff
@@ -42,13 +42,13 @@ public static void OnPlayTimeTick(Player player)
 
     public static void OnLevelUp(Player player)
     {
-        int jobId = (int) player.JobCode;
+        int jobId = (int) player.Job;
+        int level = player.Levels.Level;
         IEnumerable<TrophyMetadata> levelUpTrophies = GetRelevantTrophies(TrophyTypes.LevelUp);
         IEnumerable<TrophyMetadata> levelTrophies = GetRelevantTrophies(TrophyTypes.Level);
 
-        // TODO: This might not be right :D
         IEnumerable<TrophyMetadata> matchingTrophies = levelUpTrophies
```

Comment:
Now that I look back on this, I don't think I named these collections very well :)

Could we maybe rename `levelUpTrophies` -> `jobSpecificLevelTrophies` and `levelTrophies` to `generalLevelTrophies`? We could probably also merge the call to GetRelevantTrophies and the `.Where` filter, but this is all just personal preference :D

Reply:
I was thinking about that, i feel it would make it better to read also.